EcoSphereNetwork/CyberSpace
# Fix Black Screen and TypeScript Config

This PR fixes the black screen issue and TypeScript configuration:

### Changes

1. Fixed TypeScript Configuration:
   - Fixed tsconfig.json
   - Fixed tsconfig.node.json
   - Fixed module resolution
   - Fixed path aliases

2. Fixed Scene Components:
   - Fixed scene initialization
   - Fixed texture loading
   - Fixed mesh creation
   - Fixed scene rendering

3. Added Scene Selector:
   - Added scene switching UI
   - Added active scene indicator
   - Added smooth transitions
   - Added loading states

4. Added Sample Resources:
   - Added Earth texture
   - Added clouds texture
   - Added stars texture
   - Added network data

### Testing

1. Pull the changes:
   ```bash
   git pull origin feature/integration-fixes
   ```

2. Install dependencies:
   ```bash
   npm install
   ```

3. Start development server:
   ```bash
   npm run dev
   ```

4. Verify that:
   - Earth scene renders correctly
   - Network scene renders correctly
   - Scene switching works
   - Loading states work

### Next Steps

1. Add more scenes
2. Add more transitions
3. Add more controls
4. Add VR support
